packages/sqrl-cli-functions/tsconfig.json

```diff
@@ -1,9 +1,20 @@
 {
-  "extends": "../../tsconfig.json",
-
+  "extends": "../../tsconfig.base.json",
   "compilerOptions": {
+    "composite": true,
     "rootDir": "./src",
-    "outDir": "./lib"
+    "outDir": "./lib",
+    "tsBuildInfoFile": "lib/cache.tsbuildinfo"
   },
-  "include": ["src/**/*"]
+  "include": [
+    "src/**/*"
+  ],
+  "references": [
+    {
+      "path": "../sqrl"
+    },
+    {
+      "path": "../sqrl-common"
+    }
+  ]
```

Comment:
this array should be ideally updated with a script that checks the package’s dependency array and mirrors it here as project references. i’ve got a way of doing it in some other projects, will port it over when i get a chance.

Reply:
sure, but it's also not a big deal. these dependencies should get *very rarely* updated